cmd/config/io.go

```diff
@@ -4,25 +4,28 @@ import (
 	"errors"
 	"io/ioutil"
 	"os"
+	"path/filepath"
 
 	"gopkg.in/yaml.v3"
 )
 
 // ReadFile returns OneBuildConfiguration file content as string.
 func ReadFile() (string, error) {
-	if _, err := os.Stat(OneBuildConfigFileName); os.IsNotExist(err) {
-		return "", errors.New("no '" + OneBuildConfigFileName + "' file found in current directory")
+	oneBuildConfigFile := GetConfigFile()
+	if _, err := os.Stat(oneBuildConfigFile); os.IsNotExist(err) {
+		return "", errors.New("no '" + oneBuildConfigFile + "' file found in current directory")
```

Comment:
The error message could be different now. As you are giving the absolute path of the file in `oneBuildConfigFile`

Reply:
Ok I'll change it to
```
$ 1build -f ~/dev/1build.yaml test
no '/home/oz/dev/1build.yaml' file found
```